src/util/DataResolver.js

```diff
@@ -110,17 +108,14 @@ class DataResolver {
    * @returns {Promise<Buffer|Stream>}
    */
   static async resolveFile(resource) {
-    if (!browser && Buffer.isBuffer(resource)) return resource;
-    if (browser && resource instanceof ArrayBuffer) return Util.convertToBuffer(resource);
-    // eslint-disable-next-line no-undef
-    if (browser && resource instanceof Blob) return resource;
+    if (Buffer.isBuffer(resource)) return resource;
```

Comment:
I know it might be out of the scope of this PR, but can't this if and the next be consolidated?

Reply:
If that's done, we might as well remove the useless `else` in line 118.